samples/layouts/tile-manager/columngap/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Tile Manager Columns</title>
+  <meta charset="UTF-8" />
+
+  <link rel="shortcut icon" href="https://static.infragistics.com/xplatform/images/browsers/wc.png">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kanit&display=swap" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+  <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v8.css" type="text/css" />
+</head>
+
+<body>
+  <div id="root">
+    <div class="container sample center">
+      <div class="inputWrapper">
+      <igc-input label="Columns Number" id="rowIn" type='number' value="0" min="0"></igc-input>
+      <igc-input label="Gap Size" type='string' placeholder="10px"></igc-input>
+      <igc-input label="Minimum Column Width" placeholder="220px" type='string'></igc-input>
+      <igc-input label="Minimum Row Height" placeholder="210px" type='string'></igc-input>

```

Comment:
Changing these two settings in this sample does not really have any effect unless the viewport is resized. I would suggest we move the two inputs to the "Resizing' sample where users can observe the change during resize operations.